tensorflow/lite/CMakeLists.txt

```diff
@@ -96,6 +97,18 @@ if(${CMAKE_CROSSCOMPILING})
     message(STATUS "Pre-built 'flatc' compiler for cross-compilation purposes found: ${FLATC_BIN}")
     set(FLATBUFFERS_FLATC_EXECUTABLE "${FLATC_BIN}")
   endif()
+
+  # Setup the host Protobuf compiler.
+  set(PROTOC_PATHS
```

Comment:
I am assuming we are building a protoc within the building process? And we also don't need QNX guard for this change? Is that because it's a tf bug we want to fix?